README.md

```diff
@@ -201,6 +201,28 @@ See [here](#arguments)
 ###### Successful message
 See [here](#successful-message)
 
+#### Option 4. Use Docker image
+
+##### Step 1. Create keys and `deposit_data-*.json`
+
+Run the following command to enter the interactive CLI:
+
+```sh
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/eth2.0-deposit-cli
+```
+
+You can also run the tool with optional arguments:
+
+```sh
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/eth2.0-deposit-cli --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --folder=<YOUR_FOLDER_PATH>
+```
+
```

Comment:
Suggest adding an example of what to run for 1 validator on the Medalla testnet using English

e.g.
```
docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/eth2.0-deposit-cli --num_validators=1 --mnemonic_language=english --chain=medalla
```